Metro-Records/la-metro-dashboard
# Fix production merge host url

## Overview

See title.

Related to Metro-Records/la-metro-councilmatic#1125

### Checklist



## Testing Instructions

 * Run `blackbox_cat configs/production/.env.gpg`
 * Verify that the `MERGE_HOST` value is updated (doesn't include the string `testing`)
